google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/MavenBuildDeploymentSourceType.java

```diff
@@ -107,6 +110,11 @@ public void save(@NotNull ModuleDeploymentSource deploymentSource, @NotNull Elem
       if (deployable.getVersion() != null) {
         tag.setAttribute(VERSION_ATTRIBUTE, ((AppEngineDeployable) deploymentSource).getVersion());
       }
+
+      if (deployable.getEnvironment() != null) {
```

Comment:
I see this has been addressed. 

Noting this as "fixed" to help when scanning through this.